src/Microsoft.AspNetCore.Mvc.Localization/DependencyInjection/MvcLocalizationMvcCoreBuilderExtensions.cs

```diff
@@ -111,5 +112,218 @@ public static IMvcCoreBuilder AddViewLocalization(this IMvcCoreBuilder builder)
             MvcLocalizationServices.AddLocalizationServices(builder.Services, format, setupAction);
             return builder;
         }
+
+        /// <summary>
+        /// Adds MVC view and data annotations localization services to the application.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcCoreBuilder"/>.</param>
+        /// <returns>The <see cref="IMvcCoreBuilder"/>.</returns>
+        /// <remarks>
+        /// Adding localization also adds support for views via
+        /// <see cref="MvcViewFeaturesMvcCoreBuilderExtensions.AddViews(IMvcCoreBuilder)"/> and the Razor view engine
+        /// via <see cref="MvcRazorMvcCoreBuilderExtensions.AddRazorViewEngine(IMvcCoreBuilder)"/>.
+        /// </remarks>
```

Comment:
Please add similar `<remarks>` in the new `MvcLocalizationMvcBuilderExtensions` methods as well as the overloads below that you missed.